js/src/forum/components/UserDirectoryPage.js

```diff
@@ -38,6 +38,8 @@ export default class UserDirectoryPage extends Page {
             .filter((id) => id);
 
         this.enabledSpecialGroupFilters = [];
+
+        app.history.push('users', app.translator.trans('fof-user-directory.forum.header.back_to_user_directory_tooltip'));
```

Comment:
This would be the wrong translation, no? As going 'back' would not be back to the directory, but (likely) back to the discussion list. The label for this would be automatically generated, so I think:

```suggestion
        app.history.push('users');
```
is all that's required here